sxw.c

```diff
@@ -314,29 +311,29 @@ void SXW_Run_SOILWAT (void) {
  * 3/31/2003 - cwb - because we're always running soilwat to
  *             emulate full-size plants, computing roots etc
  *             gets done once during init plot.
+ * 8/20/2018 - CH - renamed SWA_transp to transp_resource
```

Comment:
I believe transp_resource is completely removed so we should probably remove this comment.

Reply:
Good idea.